README.md

```diff
@@ -1,79 +1,53 @@
-library having as few dependencies as possible. Having microcontrollers in mind, we want these libraries to not rely on
-a runtime — this includes language runtimes such as garbage collection, but also OS integration (which, in the case of
-Rust, is provided by its `libstd` standard library).
+## Design
 
-## Milestones
-[Milestones]: #milestones
+Bee will provide one central reference implementation of the most important
+data structures and algorithms, which will be verified and eventually
+certified.
 
-The Bee framework was primarily created to support the implementation of the IOTA
-[Coordicide](https://coordicide.iota.org/). As previously stated, on the path to Coordicide we aim to make this
-framework as open, vetted and usable as possible, by supporting the creation of tools, clients, nodes…
+By using this approach, we hope that improvements in any core components will quickly propagate to all other client libraries, rather than
+having to fix each one individually.
 
-Our proposed milestones are:
-
-1. **Fundamental crates**: Specification and implementation of the Bee fundamental crates `bee-trinary`, `bee-model`,
-and `bee-crypto`. These are the essential bricks for IOTA development.
-2. **FFI**: Foreign Function Interface to make the Bee crates available to other languages. The first project relying on
-Bee crates through FFI will be Trinity v2.
-  + We will also investigate in how far WebAssembly, `wasm`, might be an intermediate milestone.
-3. **Rust IRI**: Specification and implementation of the node-specific crates `bee-network`, `bee-tangle`, `bee-api`,
-`bee-consensus`, and `bee-gossip`. To demonstrate the modularity and robustness of the Bee framework, a node for the
-current mainnet will be implemented. Some of these crates will be repurposed for the Coordicide node.
-4. **Coordicide**: Specification and implementation of new Coordicide node-specific crates once research specifications
-are delivered.
-
-## Contributing
-[Contributing]: #contributing
+## Supporting the project
 
 If you want to discuss Bee or have some questions about it, join us on the
-[IOTA Discord server](https://discord.iota.org/) in the `#bee-dev` and `#bee-discussion` channels.
+[IOTA Discord server](https://discord.iota.org/) in the `#bee-dev` and
+`#bee-discussion` channels.
 
-If you want to contribute bug reports and pull requests or suggest new features, please see
-[`bee/CONTRIBUTING.md`](./CONTRIBUTING.md) for guidelines on how to contribute. Please note that we have a Request for
-Comments (RFC) process in place to propose, discuss, and vote on new features entering the Bee framework. You can find
-more information at [`iotaledger/bee-rfcs`](https://github.com/iotaledger/bee-rfcs/).
+If you want to be a part of development, please see the [contributing guidelines](.github/CONTRIBUTING.md) for information on how to contribute.
 
-## Why Rust?
```

Comment:
One sentence about Rust should be fine I guess. I would propose:
For the Bee framework the Rust programming language was chosen, because of its C/C++ like performance and its strong memory safety guarantees. You can learn more about Rust [here](https://www.rust-lang.org).

Reply:
Just realized those are 2 sentences. Oopsie!